Lib/email/_header_value_parser.py

```diff
@@ -505,6 +505,12 @@ class DotAtomText(TokenList):
     as_ew_allowed = True
 
 
+class NoFoldLiteral(TokenList):
+
```

Comment:
I think will be better remove this unused line.

And do the same with the other methods.

Reply:
Agreed.

Reply:
I just wanted to make it look consistent with the other classes above. All of them have an empty line.

Reply:
I'd be fine with deleting all those blank lines, but I put them in because at the time I thought that's what PEP 8 recommended, and some of the TokenList subclasses do have additional methods and in that case I think it looks better.

Reply:
Done! Removed empty lines in all the classes that don't have a method.